resources_pages/install_ds_stack_ubuntu.md

```diff
@@ -318,26 +283,27 @@ Sometimes a kernel loads, but doesn't work as expected. To test whether your ins
 
 ## LaTeX
 
-We will install the lightest possible version of LaTeX and it's necessary packages as possible so that we can render Jupyter notebooks and R Markdown documents to html and PDF.
+We will install the lightest possible version of LaTeX and it's necessary packages as possible so that we can render Jupyter notebooks and R Markdown documents to html and PDF. If you have previously installed LaTeX, please uninstall it before proceeding with these instructions.
 
-First, open RStudio and run the following commands to install the tinytex package and install tinytex:
+First, open RStudio and run the following commands to install the tinytex package and setup tinytex:
 
 ```
 install.packages('tinytex')
 tinytex::install_tinytex()
 ```
 
-The above is all we need to have LaTeX work with R Markdown documents, however to for Jupyter we need to add the TinyTex executables to our `PATH` and install several more packages. 
+The above is all we need to have LaTeX work with R Markdown documents, however to for Jupyter we need to add the TinyTex executables to our `PATH` and install several more packages.
 
 To add the TinyTex executables to our `PATH` we need to edit our `.bashrc` file. The TinyTex executables are usually installed in `$HOME/bin`. Thus, add the lines below to the bottom of your `.bashrc` file (which you can open by `code $HOME/.bashrc`):
 
 ```
 # add TinyTex executables to the path
 export PATH="$HOME/bin:$PATH"
```

Comment:
Is it safe to prepend this directory or should we append it instead? I don't know what is higher risk, unwanted executables in this dir that get unwanted preference over other system packages, or unintentional addition of latex packages to other path dirs that get preference over tinyTex unless it is first.

Reply:
Sure, change it to append instead (assuming it still works, which it should). I wondered that as I was doing it...

Also, can you confirm that after these LaTeX install instructions you can create PDFs from Jupyter?

Reply:
Great!